site/en/docs/extensions/reference/printing/index.md

```diff
@@ -2,4 +2,95 @@
 api: printing
 ---
 
-<!-- Intentionally blank -->
+## Manifest
+
+All `chrome.printing` methods and events require you to declare the "printing" permission in the [extension manifest](/docs/extensions/mv3/manifest/). For example:
+
+```json
+{
+  "name": "My extension",
+  ...
+  "permissions": [
+    "printing"
+  ],
+  ...
+}
+```
+
+## Examples
+
+The examples below demonstrate using each of the methods in the printing namespace. This code is copied from or based on the [printing](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/api-samples/printing) in the Extension samples database.
+
+### cancelJob()
+
+This example uses the `onJobStatusChanged` handler to hide a 'cancel' button when the `jobStatus` is neither `PENDING` or `IN_PROGRESS`. Note that on some networks or when a Chromebook is connected directly to the printer, these states may pass too quickly for the cancel button to be visible long enough to be called. This is greatly simplified the printing example.
+
```

Comment:
Should we add [labels](https://developer.chrome.com/docs/handbook/components/#labels) to the code samples?

Reply:
The label would be 'printers.js', which would need to be explained. I realize that I could say that about sw, content, and popup, but those are standard and it solves itself over time. 'printers.js' is completely unique to this page, so I'm not sure what the label gets us. If you actually go to the sample, it's pretty obvious where to look, even if you didn't know ahead of time.

(I've been thinking for a few days that we should have a meeting about this. Maybe we don't need labels everywhere, but we should have a common heuristic.)